src/themes/legacy.scss

```diff
@@ -45,7 +45,7 @@
 
 // temporary logic for legacy
 .wxc-member-roster .title { display: none; }
-.wxc-banner.my-preview { display: none; }
+.wxc-banner.preview-banner { display: none; }
```

Comment:
you can use the parent to identify the banner, eg `.wxc-settings .wxc-banner` or `.wxc-interstitial-meeting .wxc-banner`